### PR TITLE
spotify: 1.0.70.399.g5ffabd56-26 -> 1.0.72.117.g6bd7cc73-35

### PR DESCRIPTION
**Pull request by sternenseemann**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/audio/spotify/default.nix [hunk 1]
```diff
@@ -9,7 +9,7 @@ let
   # Latest version number can be found at:
   # http://repository-origin.spotify.com/pool/non-free/s/spotify-client/
   # Be careful not to pick the testing version.
-  version = "1.0.70.399.g5ffabd56-26";
+  version = "1.0.72.117.g6bd7cc73-35";
```
Comment (Mic92) [inline pkgs/applications/audio/spotify/default.nix]: Is this the latest stable or is this a testing version?

Comment (sternenseemann) [reply pkgs/applications/audio/spotify/default.nix]: probably it's the testing version, the older one is always the stable one, is it?

**Comment by globin** [comment]
Duplicate of #36161 with the same problem as mentioned there

**Comment by sternenseemann** [comment]
@globin rebased to use 1.0.69 <https://github.com/sternenseemann/nixpkgs/commit/7d7d25fb3b3a79e49211e98320f7da8a2bacab21>

**Pull request closed without merging**